tim1/src/main/java/isamrs/tim1/service/DestinationService.java

```diff
@@ -0,0 +1,48 @@
+package isamrs.tim1.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import isamrs.tim1.dto.DestinationDTO;
+import isamrs.tim1.dto.MessageDTO;
+import isamrs.tim1.model.Airline;
+import isamrs.tim1.model.Destination;
+import isamrs.tim1.model.Location;
+import isamrs.tim1.repository.DestinationRepository;
+import isamrs.tim1.repository.LocationRepository;
+import isamrs.tim1.repository.ServiceRepository;
+
+@Service
+public class DestinationService {
+
+	@Autowired
+	DestinationRepository destinationRepository;
+	
+	@Autowired
+	ServiceRepository serviceRepository;
+	
+	@Autowired
+	LocationRepository locationRepository;
+
+	public ResponseEntity<MessageDTO> addDestination(DestinationDTO destDTO) {
+	    Airline a = (Airline) serviceRepository.findOneByName(destDTO.getAirlineName());
+	    if (a == null)
+			return new ResponseEntity<MessageDTO>(new MessageDTO("Airline does not exist.", ""), HttpStatus.BAD_REQUEST);
+	    Destination dest = destinationRepository.findOneByName(destDTO.getNameOfDest());
+	    if (dest != null)
+	    	return new ResponseEntity<MessageDTO>(new MessageDTO("Destination with same name already exists.", ""), HttpStatus.OK);
+		Destination d = new Destination();
```

Comment:
Fix indentation

Reply:
Indentation is good in the latest version, I do not know why Github shows this